README.md

```diff
@@ -1,13 +1,26 @@
-# Tonic Research Project Template
+# Project title
 
-This project provides a template structure for research projects within the CRC/TRR 135 according to the results of a joint work with CRC 1158 and CRC 1315 (see https://genr.eu/wp/towards-a-standardized-research-folder-structure/) for details.
+## Abstract:
 
-# How to create a new study
+## Contributors :
 
-Creating a new study project according to the tonic template is really easy: Just use the button `Use this template`, specify the owner, name and privacy and select button `Create repository from template`. Here we go!
+Corresponding author(s): 
 
-# DMP tools
-*  [DMPonline](https://dmponline.dcc.ac.uk/) - A tool developed by the Digital Curation Center (UK). It covers the requirements of EU Horizon 2020 and the major funding organizations in the UK.
-*  [DMPTool](http://dmptool.org/) - A tool initiated by the University of California Digital Curation Center. It focuses accordingly on the most important US funding organizations.
-*  [GFBio Data Management Plan Tool](https://www.gfbio.org/plan) - GFBio provides its users in the field of biodiversity and environmental research (but not exclusively) with this tool. It contains basic questions about data management planning and documents the given information in a structured way.
-*  [Research Data Management Organiser (RDMO)](https://rdmorganiser.github.io/) - A tool to plan, implement and manage the research data management of a research project. The tool was developed as part of a DFG-funded project with many stakeholders. First facilities use it productively.
\ No newline at end of file
+- jane doe, jane.doe@example.com, https://orcid.org/XXXX-XXXX-XXXX-XXXX
```

Comment:
If find this unnecessary, markdown can handle it, it makes it not so yaml like

Reply:
This is a matter of portability, sustainability, and reducing ambiguity. It's not _markdown_ according to the [CommonMark specification][commonmark-autolinks] that can handle it, but Github's Markdown implementation. Recognizing bare URLs without pointy brackets in text blocks as links is not part of CommonMark, but an extension by [Github flavored markdown][gfm-autolinks] (GFM). In [Pandoc's Markdown][pandoc-markdown], e.g., it is disabled by default.

[CommonMark][commonmark-spec] is a widely adopted Markdown specification which can be parsed and generated by [many tools][commonmark-implementations]. [GFM][gfm] is a superset of CommonMark. We don't know where and in which contexts the files in our template will be used, so we should adhere to established standards as much as possible instead of relying on domain-specific implementation details. Analogously, one can write a web app for a specific browser and rely on its features, or one can use web standards that _should_ be adopted by most browsers.

For highly domain-specific content which is written for a determined tool or context, it can make much sense to use language extensions. This could be the conversations in Github's issues and pull requests, or maybe Pandoc documents. Pandoc's Markdown combined with bookdown introduces a bunch of extensions for academic writing. Most of these extensions are useful, because they add new semantic concepts instead of just syntax sugar making things more implicit and casing more confusion. But it must be taken into account that a document using these extensions is bound to Pandoc, and if the advantages or convenience is worth the loss of portability.

[commonmark-spec]: https://spec.commonmark.org
[commonmark-autolinks]: https://spec.commonmark.org/0.30/#autolinks
[commonmark-implementations]: https://github.com/commonmark/commonmark-spec/wiki/List-of-CommonMark-Implementations
[pandoc-markdown]: https://pandoc.org/MANUAL.html#pandocs-markdown
[gfm]: https://github.github.com/gfm/
[gfm-autolinks]: https://github.github.com/gfm/#autolinks-extension-